sys/mutex.c

```diff
@@ -45,3 +46,7 @@ void mtx_unlock(mtx_t *mtx) {
   turnstile_signal(&mtx->turnstile);
   cs_leave();
 }
+
+bool mtx_is_locked(mtx_t *mtx) {
```

Comment:
This function isn't useful by itself, but I found it convenient to use it for some assertions. After cleaning the code, it turns out I only use it once, so I can remove it, if you want - but we may find it useful in other places as well.